### PR TITLE
Pull request with main changes

### PR DESCRIPTION
**Pull request by c-giordano**
I think this is how you do it?

### CODE REVIEW

File: pythonStacker/plotHistogramsRedo.py [hunk 1]
```diff
@@ -31,8 +31,12 @@ def parse_arguments():
 
     parser.add_argument("--EFT_ratio", dest="EFT_ratio", action="store_true", default=False)
     parser.add_argument("--EFT_fullbkg", dest="EFT_fullbkg", action="store_true", default=False)
+    parser.add_argument("--BSM_ratio", dest="BSM_ratio", action="store_true", default=False)
+    parser.add_argument("--BSM_fullbkg", dest="BSM_fullbkg", action="store_true", default=False)
     parser.add_argument("--SBRatio", dest="SBRatio", action="store_true", default=False)
     parser.add_argument("--shapes", dest="shapes", action="store_true", default=False)
+    parser.add_argument("--hasBSM", action="store_true", default=True)
```
Comment (NielsVdBossche) [inline pythonStacker/plotHistogramsRedo.py:38]: Is this flag needed?

File: pythonStacker/plotHistogramsRedo.py [hunk 2]
```diff
@@ -240,12 +256,16 @@ def get_lumi(years):
         "2016": 36.3,
         "2017": 41.5,
         "2018": 59.8,
+        # "Run2": 167.0,
     }
+    sum_years = {"2016", "2017", "2018"}
     for year in years:
-        total_lumi += lumi[year]
+        if year in sum_years:
+            total_lumi += lumi[year]
```
Comment (NielsVdBossche) [inline pythonStacker/plotHistogramsRedo.py:264]: Why are these lines added?

File: pythonStacker/createBSMWeights.py [hunk 3]
```diff
@@ -20,7 +21,7 @@ def parse_arguments():
                         help="which event class to check")
     parser.add_argument("-p", "--process", dest="process", action="store", default="TTTT_BSM")
     parser.add_argument("--storage", dest="storage", type=str,
-                        default="Intermediate", help="Path at which the histograms are stored")
+                        default="Intermediate_VO", help="Path at which the histograms are stored")
```
Comment (NielsVdBossche) [inline pythonStacker/createBSMWeights.py:24]: Slightly worrying for other things down the line, why is this done? Cause everything down the line just uses the generic storage argument afaik, so this just raises questions

**Review by NielsVdBossche — CHANGES_REQUESTED** [review]

File: pythonStacker/plotHistogramsRedo.py (hunk 1 above)
Comment (c-giordano) [reply pythonStacker/plotHistogramsRedo.py:38]: I use it to specify the output subdirectory in order not to have too many plots in the same one, not strictly necessary but useful for organization reasons

File: pythonStacker/plotHistogramsRedo.py (hunk 2 above)
Comment (c-giordano) [reply pythonStacker/plotHistogramsRedo.py:264]: To fix the value of the Run 2 cross-section (was summing 2016, 2016preVFP, and 2016postVFP), it is indeed contrite but I needed a quick solution

File: pythonStacker/createBSMWeights.py (hunk 3 above)
Comment (c-giordano) [reply pythonStacker/createBSMWeights.py:24]: Because I store each model in a different Intermediate_<> folder; at the beginning each one of them had its own problems and I wanted to keep it separate as much as possible. Still not needed for everybody, it's a bookkeeping solution that worked for me

File: pythonStacker/plotHistogramsRedo.py (hunk 1 above)
Comment (NielsVdBossche) [reply pythonStacker/plotHistogramsRedo.py:38]: ok but can't the existing args.UseBSM be used instead? just to remove additional arguments that don't really do anything

File: pythonStacker/plotHistogramsRedo.py (hunk 2 above)
Comment (NielsVdBossche) [reply pythonStacker/plotHistogramsRedo.py:264]: ok but that means the 3 2016's are loaded so that you're plotting the wrong things. So this shouldn't be done, instead you need to make sure you just specify -y 2016 2017 2018 or with 2016 split. The lumi being wrong is just the signal to check and modify this
